notebooks/jacques_drivendata_map_floodwater_benchmark_tutorial.ipynb

```diff
@@ -78,8 +78,10 @@
         }
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #11.    from pytorch_lightning import seed_everything
</pre><p>Nit: Good work moving this at the top of the notebook! You should add this below the pytorch import, it will be grouped more nicely. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/JayThibs/map-floodwater-satellite-imagery/pull/14/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>